### DISCUSSION TITLE
3.0 vs 2.11

### DISCUSSION THREAD

**Discussion by mikedizon** [saleor/saleor#7115] [Q&A] [answered]
Just getting started with this project. What is the consensus for using the latest stable (2.11 as of this writing) vs the latest from the master branch?

Will there be lots of breaking changes between major releases?

**Comment by krzysztofwolski** [comment, marked as answer]
There are big changes in 3.0:
- channels ( https://docs.saleor.io/docs/3.0/developer/channels )
- new attributes 
- new editor for rich text (pages, product descriptions, etc)
- many bugfixes (if possible we backported those also to 2.11)
- dropped deprecated fields from 2.11
- better, more flexible hooks and apps/plugins
- more possibilities with metadata
- more robust warehousing system
and many more. Also, we improved our documentation on 3.0 branch so it's easier to start.
ATM we are focused more on stability and performance. There will be a few new features added, but don't think that API will change a lot on 3.0 branch.

If I would start a new project with a release date >=2months, I would choose 3.0. Especially if you want to maintain a fork and introduce any changes - apps and plugins will allow you to easily update with the upstream. 

If you have already chosen 2.11, there will be a migration guide. Changes will be needed mostly on storefront side - data will be migrated automatically and multichannel won't affect you unless you create more channels (if you have only one channel in the system, queries do not require passing its slug).

**Reply by muhammadarslan** [reply]
when 3.1 will be available? it says Q3, seems like in few days?

**Reply by muhammadarslan** [reply]
when 3.1 will be available? it says Q3, seems like in few days?
